input/pagecontent/3-6-3-DiagnosticReport-update.md

```diff
@@ -36,7 +36,7 @@ Request Method | Operation
 
 #### Number of Entries in Transaction Bundle
 
-FHIRcast doesn't prescribe a limit on the number of entries in the transaction Bundle; however, implementers should expect a limit for production-grade software. Generally, the upper range of entries in a FHIRcast transaction bundle is in the *dozens*. Recipients SHOULD return an error when they receive a FHIRcast event notification that is too large to support. Specifically, recipients SHALL either synchronously return an HTTP error status of [HTTP 413 - Content Too Large](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413), or asynchronously a `syncerror` with an `OperationOutcome.issue.code` = "[too-long](https://hl7.org/fhir/R4/valueset-issue-type.html)".
+Refer to Number of Entries in Transaction Bundle in [Content Sharing](https://hl7.org/fhir/uv/fhircast/2024May/2-10-ContentSharing.html) for more information.

```

Comment:
```suggestion
Refer to Number of Entries in Transaction Bundle in [Content Sharing](2-10-ContentSharing.html) for more information.

```
Noura - be careful with absolute urls. Once we publish this spec as STU3, this hyperlink would've kept a link to the balloted version (at which you're new section doesn't exist). 

One should probably never hyperlink an IG to itself with an absolute url.